package.json

```diff
@@ -94,6 +94,9 @@
     "type": "git",
     "url": "git@github.com:hypertrace/hyperdash-angular.git"
   },
+  "publishConfig": {
+    "registry": "https://us-npm.pkg.dev/hypertrace-repos/npm-publish"
```

Comment:
The [docs](https://github.com/semantic-release/npm?tab=readme-ov-file#npm-configuration) here are wildly confusing. They state
> The [registry](https://docs.npmjs.com/misc/registry) can be configured via the npm environment variable NPM_CONFIG_REGISTRY and will take precedence over the configuration in .npmrc.

which isn't what we saw. Later in the docs, it states:
> Notes:

> The presence of an .npmrc file will override any specified environment variables.

So trying a different approach to split the read and publish registries.